Systems/computers.xml

```diff
@@ -0,0 +1,573 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<PropertyList> 
+
+<!-- INS computer -->
+
+  <filter>
+        <type>gain</type>
+        <gain>1.0</gain>
+         <enable>
+         <condition> 
+          <equals>
+          <property>sim/failure-manager/systems/computers/INS/heading/failure-level</property>
+          <value>0</value>
+         </equals>  
+        </condition>
+       </enable>
+       <input>
+       <property>orientation/heading-deg</property> 
+      </input>
+     <output>systems/computers/INS/heading</output>
+  </filter>
+
+  <filter>
+        <type>gain</type>
+        <gain>1.0</gain>
+         <enable>
+         <condition> 
+          <equals>
+          <property>sim/failure-manager/systems/computers/INS/pitch/failure-level</property>
+          <value>0</value>
+         </equals>  
+        </condition>
+       </enable>
+       <input>
+       <property>orientation/pitch-deg</property> 
+      </input>
+     <output>systems/computers/INS/pitch</output>
+  </filter>
+
+  <filter>
+        <type>gain</type>
+        <gain>1.0</gain>
+         <enable>
+         <condition> 
+          <equals>
+          <property>sim/failure-manager/systems/computers/INS/roll/failure-level</property>
+          <value>0</value>
+         </equals>  
+        </condition>
+       </enable>
+       <input>
+       <property>orientation/roll-deg</property> 
```

Comment:
Maybe this should be instrumentation/attitude-indicator/indicated-roll-deg instead?

Reply:
Yeah. If we REALLY wanted to dive in to this, we'd simulate every single computer and have properties that did the calculations and spat out an output (I guess some things such as autopilot is in fact this). However the way it's working at the moment is that, those 8 or however many computers, are all simulated by the one each of us are sat in front of! But yes, I think this should change. I have already started the process with the ADC's and GSS's